packages/app/app/components/PlayQueue/QueuePopup/index.js

```diff
@@ -49,6 +51,11 @@ export const QueuePopup = ({
     actions.changeTrackStream(track, stream);
   };
 
+  const handleCopyStreamUrl = useCallback(() => {
```

Comment:
This callback should be defined in a container, we want to avoid complex dependencies like Electron in components.